tests/bulk_create/tests.py

```diff
@@ -234,10 +234,23 @@ def test_bulk_insert_expressions(self):
 
     @skipUnlessDBFeature('has_bulk_insert')
     def test_bulk_insert_nullable_fields(self):
+        auto_field_instance, _ = AutoFieldModel.objects.get_or_create()
+        small_auto_field_instance, _ = SmallAutoFieldModel.objects.get_or_create()
+        big_auto_field_instance, _ = BigAutoFieldModel.objects.get_or_create()
+        auto_fields = {
```

Comment:
Is this good for you or would you rather these values as default in model definition ?